Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json

```diff
@@ -135,8 +135,8 @@
         },
         {
             "style": "Text",
-            "text": "Superalgos uses CCXT (CryptoCurrency eXchange Trading Library) to connect to exchanges. It supports hundreds of exchanges to which it connects through APIs created on the exchange by the user. One of the best feature of ccxt is that it is exchange agnostic. Whether you trade on FTX or Bybit the process to connect to the exchange is the same. The library accepts many parameters to fine tune the connection when needed. For well supported exchanges like Binance, all you need to connect to the exchange is to declare the codeName that is the name of the exchane you want to use, usually in lowercase letters.",
-            "updated": 1650647816973
+            "text": "Superalgos uses CCXT (CryptoCurrency eXchange Trading Library) to connect to exchanges. It supports hundreds of exchanges to which it connects through APIs created on the exchange by the user. One of the best feature of ccxt is that it is exchange agnostic. Whether you trade on FTX or Bybit the process to connect to the exchange is the same. The library accepts many parameters to fine tune the connection when needed. For well supported exchanges like Binance, all you need to connect to the exchange is to declare the codeName that is the name of the exchage you want to use, usually in lowercase letters.",
+            "updated": 1650806161770
```

Comment:
You missed a 'n' in exchange, as you fixed the missing 'g'. :+1:

Reply:
I check this and update the PR